ui-modules/blueprint-composer/app/components/spec-editor/spec-editor.directive.js

```diff
@@ -39,6 +39,11 @@ const SUBSECTION = {
     PARAMETERS: 'parameters'
 }
 
+export const SUBSECTION_TEMPLATE_URL = {
```

Comment:
is there any way we can allow a downstream project to update the template cache directly, without downstream-only constants being necessary here?

Reply:
I do not have an answer at this time. Open to suggestions.